DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json

```diff
@@ -3047,7 +3081,7 @@
                                     "Position": "1"
                                 },
                                 "ContentPickerFieldSettings": {
-                                    "Hint": "Creates relationships of type [:relatedOccupation].",
+                                    "Hint": "[\"#Skills\", \"#WhatItTakes_Content\"]",
```

Comment:
We actually pick up the name of the relationship to create out of the hint, so this will change what relationship we create (it will fall-back to a default).